File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
t = int(input())

for i in range(t):
    h, w, n = map(int, input().split())
    r1 = n%h
    r2= n//h + 1
    
    if r1 == 0 :
        r1=h
        r2=n//h
        print(r1*100+r2)
    else:
        print(r1*100+r2)

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.js
var input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
var T = parseInt(input[0]);
var room;
for(var i=1; i<T+1; i++) {
    var hotel = input[i].split(' ').map(x => Number(x));
    var h = hotel[2] % hotel[0];
    var w = parseInt(hotel[2] / hotel[0]) + 1;

    if (h === 0) {
        room = hotel[0]*100 + w-1;
    }else {
        room = h*100 + w;
    }
    console.log(room);
}
